src/spi/Adafruit_FlashTransport_SPI.cpp

```diff
@@ -91,13 +91,13 @@ bool Adafruit_FlashTransport_SPI::writeCommand(uint8_t command,
 }
 
 bool Adafruit_FlashTransport_SPI::eraseCommand(uint8_t command,
-                                               uint32_t address) {
+                                               uint32_t addr) {
   beginTransaction(_clock_wr);
 
-  uint8_t cmd_with_addr[] = {command, (address >> 16) & 0xFF,
-                             (address >> 8) & 0xFF, address & 0xFF};
+  uint8_t cmd_with_addr[5] = {command};
+  fillAddress(cmd_with_addr + 1, addr);
```

Comment:
right, I forgot the erase command :), since the 8KB fram doesn't need this to work.